From 04cbfcc5e0d1edf4809daa5a306910395cf536b6 Mon Sep 17 00:00:00 2001
From: Rakesh Pallerla <rakesh@codeaurora.org>
Date: Fri, 26 Sep 2014 16:13:46 +0530
Subject: [PATCH 08/13] Store Network Mode selected in subId Table.

Modify code to store network selected by user in
subId table.
DELETED:telephony/java/android/telephony/SubInfoRecord.java

Change-Id: I361c5cbc47ea6079deb3e807276a3f3504a9535e
---
 telephony/java/android/telephony/SubscriptionManager.java | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/telephony/java/android/telephony/SubscriptionManager.java b/telephony/java/android/telephony/SubscriptionManager.java
index 88c5fdafad9..3ed638cc156 100644
--- a/telephony/java/android/telephony/SubscriptionManager.java
+++ b/telephony/java/android/telephony/SubscriptionManager.java
@@ -294,6 +294,9 @@ public class SubscriptionManager {
      */
     public static final String USER_NETWORK_MODE = "user_network_mode";
 
+    /** @hide */
+    public static final int DEFAULT_NW_MODE = -1;
+
     /**
      *  TelephonyProvider column name for extreme threat in CB settings
      * @hide
-- 
2.17.0

